README.md

```diff
@@ -1,24 +1,279 @@
 
-* System dependencies
+## Definitions
+#### External Term
+Used when caching a value/URI pair from an external controlled vocabulary within URI Service. Example: We want to add an entry for U.S. President Abraham Lincoln to our URI Service datastore, so we'll create an external term that references his Library of Congress URI.
 
-* Configuration
+#### Local Term
+Used when defining locally-managed terms in the URI Service. Automatically creates a local URI for a new local term. Example: We want to maintain a vocabulary for various departments within a university, and we want to create locally-managed URIs for these departments.
 
-* Database creation
+#### Temporary Term
+Used when you want to add a value to your URI Service datastore, but have no authority information about the term or do not wish to create a local URI. No two temporary terms within the same vocabulary can have the same value. Basically, a temporary term is intended to identify an exact string value rather than identifying an intellectual entity. Temporary terms should eventually be replaced by external or local terms later on when more information is known about the entity to which you are referring . Example: We want to record information about the author of an old and mysterious letter by "John Smith." We don't know which "John Smith," this refers to, so we'll create (or re-use) a temporary URI that's associated with the value "John Smith."
 
-* Database initialization
+#### Custom Fields
+TODO
 
-* How to run the test suite
+## Installation
+TODO
 
-* Services (job queues, cache servers, search engines, etc.)
+## Services (job queues, cache servers, search engines, etc.)
+URI Services stores local, temporary and external terms in a mysql database. Solr is used to index terms for fast lookup and searching. Redis/Resque will be used to run scheduled jobs.
 
-* Deployment instructions
+## Deployment instructions
+TODO
 
-* ...
+## API v1
+
+### Vocabularies
+#### List Vocabularies
+- **Request**
+  `GET /vocabularies`
+
+- **Success Response**
+
+  Sorts vocabularies alphabetically by label. Defaults to `page = 1` and `per_page = 20`.
+
+  _Status:_ 200
+
+  _Body:_
+
+  ```json
+  {
+    "page": 1,
+    "per_page": 10,
+    "total_results": 2,
+    "vocabularies": [
+      { "string_key": "name" , "label": "Names" },
+      { "string_key": "collections" , "label": "Collections" }
+    ]
+  }
+  ```
+
+#### Find Single Vocabulary
+- **Request**
+
+  `GET /vocabularies/:string_key`
+
+- **Successful Response**
+
+  _Status:_ 200
+
+  _Body:_
+  ```json
+  {
+     "string_key": "names",
+     "label": "Names"
+  }
+  ```
+
+#### Create Vocabulary
+- **Request**
+
+  `POST /vocabularies`
+
+  `string_key` is a required parameter.
+
+- **Success Response**
+
+  _Status:_ `201`
+
+  _Body:_
+    ```json
+    {
+      "string_key": "names",
+      "label": "Names"
+    }
+    ```
+
+#### Update Vocabulary
```

Comment:
Would be good to mention here that the only property of a vocabulary that you're allowed to update is the `label`.